client/js/components/header.js

```diff
@@ -0,0 +1,174 @@
+import showHide from './show-hide';
+import { nodeList, KEYS } from './../util';
+
+const header = (el) => {
+  const headerNav = el.querySelector('.js-header-nav');
+  const headerLower = el.querySelector('.js-header-lower');
+  const headerSearchForm = el.querySelector('.js-search');
+  const headerSearchInput = headerSearchForm.querySelector('.js-search-input');
+  const headerItems = headerNav.querySelectorAll('.js-show-hide');
+  let searchToggle;
+  let dropdowns;
```

Comment:
Where are we using dropdowns?

Unless we have a really good solution to touch devices, I'd argue we should come up with a better solution instead of walking ourselves into a corner with an antipattern.